src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -13082,8 +13082,10 @@ public static void Main(string[] args)
                 Diagnostic(ErrorCode.ERR_ArrayInitializerIncorrectLength, "{}").WithArguments("3").WithLocation(7, 31));
         }
 
-        [Fact]
-        public void CS0853ERR_ExpressionTreeContainsNamedArgument01()
+        [Theory]
```

Comment:
>[Theory] [](http://example.com/codeflow?start=8&length=8)

It looks like all comments mage in CodeGenExprLambdaTests are applicable to changes in this file too.